sdk/eventgrid/eventgrid/review/eventgrid.api.md

```diff
@@ -10,6 +10,117 @@ import { KeyCredential } from '@azure/core-auth';
 import { OperationOptions } from '@azure/core-http';
 import { PipelineOptions } from '@azure/core-http';
 
+// @public
+export interface ACSChatEventBaseProperties {
+    recipientId?: string;
+    threadId?: string;
+    transactionId?: string;
+}
+
+// @public
+export type ACSChatMemberAddedToThreadWithUserEventData = ACSChatThreadEventBaseProperties & {
+    time?: Date;
+    addedBy?: string;
+    memberAdded?: ACSChatThreadMemberProperties;
+};
+
+// @public
+export type ACSChatMemberRemovedFromThreadWithUserEventData = ACSChatThreadEventBaseProperties & {
+    time?: Date;
+    removedBy?: string;
+    memberRemoved?: ACSChatThreadMemberProperties;
+};
+
+// @public
+export type ACSChatMessageDeletedEventData = ACSChatMessageEventBaseProperties & {
+    deleteTime?: Date;
+};
+
+// @public
+export type ACSChatMessageEditedEventData = ACSChatMessageEventBaseProperties & {
+    messageBody?: string;
+    editTime?: Date;
+};
+
+// @public
+export type ACSChatMessageEventBaseProperties = ACSChatEventBaseProperties & {
+    messageId?: string;
+    senderId?: string;
+    senderDisplayName?: string;
+    composeTime?: Date;
+    type?: string;
+    version?: number;
+};
+
+// @public
+export type ACSChatMessageReceivedEventData = ACSChatMessageEventBaseProperties & {
+    messageBody?: string;
+};
+
+// @public
+export type ACSChatThreadCreatedWithUserEventData = ACSChatThreadEventBaseProperties & {
+    createdBy?: string;
+    properties?: {
+        [propertyName: string]: any;
+    };
+    members?: ACSChatThreadMemberProperties[];
+};
+
+// @public
+export type ACSChatThreadEventBaseProperties = ACSChatEventBaseProperties & {
+    createTime?: Date;
+    version?: number;
+};
+
+// @public
+export interface ACSChatThreadMemberProperties {
```

Comment:
MemberProperties suffix seems a bit strange, too, since interfaces by definition define member properties 😁

Reply:
This is now called ACSChatThreadMember, because it describes the shape of a member in a chat, the redudent properties suffix has been removed.